Xamarin.Forms.Platform.UAP/Properties/AssemblyInfo.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using Xamarin.Forms;
+using Xamarin.Forms.Platform.UAP;
 using Xamarin.Forms.Platform.UWP;
 
 [assembly: Dependency(typeof(WindowsSerializer))]
 
 // Views
 
 [assembly: ExportRenderer(typeof(Layout), typeof(LayoutRenderer))]
-[assembly: ExportRenderer(typeof(BoxView), typeof(BoxViewRenderer))]
+[assembly: ExportRenderer(typeof(BoxView), typeof(BoxViewRenderer2))]
```

Comment:
I think we are not going to make this the default renderer from the start. IMO there would need to be some kind of transition period. People can use "BoxViewRenderer2" if they need it and we give people the time to transition become of the obsoleting message. Later we will switch to this new one.

Reply:
Sounds good to me